Dockerfile

```diff
@@ -58,7 +58,8 @@ EXPOSE 3000
 WORKDIR /app
 
 RUN apk update && apk --no-cache --quiet add --update \
-    dumb-init
+    dumb-init \
+    git
```

Comment:
Since `git` is a runtime dependency, I think this should just go above into the `base` stage (and removed from the `builder` stage).

I'm not that familiar w/these new multi-stage builds though so up to you.